### PR TITLE
add createAccount test

### PR DESCRIPTION
**Pull request by vdemnyk**
Created 'Create account successful' test.

### CODE REVIEW

File: pages/login.page.js [hunk 1]
```diff
@@ -44,6 +46,16 @@ class LoginPage extends WebPage {
         return await this.getAlertBlock().getText();
     }
 
+    async typeEmailSignup(email) {
+        await this.getEmailSignupFieldElement().sendKeys(email);
```
Comment (nbabii) [inline pages/login.page.js]: Add allure step here, like `entering email ${email}`

File: pages/register.page.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+let phoneFieldLocator = by.css('#phone_mobile');
+let registerBtnLocator = by.css('#submitAccount');
+
+
+class RegisterPage extends WebPage {
+    constructor() {
+        super();
+    }
+
+    async typeFirstName(firstName) {
```
Comment (nbabii) [inline pages/register.page.js]: create single method for first name and last name, add allure step there

File: pages/register.page.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        await this.getFirstNameElement().sendKeys(firstName);
+    }
+
+    async typeLastName(lastName) {
+        await this.getLastNameElement().sendKeys(lastName);
+    }
+
+    async typePassword(password) {
```
Comment (nbabii) [inline pages/register.page.js]: add a password to the method above (for the first name and lastname)

File: pages/register.page.js [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+        await this.getPasswordElement().sendKeys(password);
+    }
+
+    async typeAddress(address) {
```
Comment (nbabii) [inline pages/register.page.js]: create a single method for address, state, city, postcode and add allure step there

File: pages/register.page.js [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+    async selectState() {
+        await this.getStateDropdownElement().click();
+        await this.getStateItemElement().click();
+    }
+
+    async typePostcode(postcode) {
+        await this.getPostcodeElement().sendKeys(postcode);
+    }
+
+    async typePhone(phone) {
```
Comment (nbabii) [inline pages/register.page.js:43]: add allure step for phone number

**Review by nbabii — CHANGES_REQUESTED** [review]
good work, please check few comments

File: pages/login.page.js (hunk 1 above)
Comment (vdemnyk) [reply pages/login.page.js]: Done

File: pages/register.page.js (hunk 2 above)
Comment (vdemnyk) [reply pages/register.page.js]: Done

File: pages/register.page.js (hunk 3 above)
Comment (vdemnyk) [reply pages/register.page.js]: Done

File: pages/register.page.js (hunk 4 above)
Comment (vdemnyk) [reply pages/register.page.js]: Done

File: pages/register.page.js (hunk 5 above)
Comment (vdemnyk) [reply pages/register.page.js:43]: Done

**Review by nbabii — CHANGES_REQUESTED** [review]
Add values to be entered into the allure ateps

**Pull request merged**